modules/nf-core/bismark/summary/main.nf

```diff
@@ -7,7 +7,7 @@ process BISMARK_SUMMARY {
         'quay.io/biocontainers/bismark:0.24.0--hdfd78af_0' }"
 
     input:
-    path(bam)
+    val(bam)
```

Comment:
the one time I like val channels for modules 😆

Reply:
that one little change can probably save a **lot** of time (and money). To be honest though, I don't quite understand how a value channel would then be picked up the the `glob *.bam` that takes place in the work folder?

Reply:
There is no glob? I updated the command so that it is given each explicit BAM filename. eg:

```
bismark_summary bam_1.bam bam2.bam bam3.bam
```